src/AppBundle/Form/ProcurationElectionsType.php

```diff
@@ -19,15 +19,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('electionPresidentialFirstRound', CheckboxType::class, [
                 'required' => false,
+                'data' => false,
```

Comment:
It's not here it should be modified then, but in the entity itself: https://github.com/EnMarche/en-marche.fr/blob/master/src/AppBundle/Entity/ProcurationRequest.php#L219